modules/mediawiki/files/cookbooks/rename-wiki.py

```diff
@@ -0,0 +1,53 @@
+import subprocess
+import argparse
+
+# Define the mapping of db clusters to db names
+db_clusters = {
+    'c1': 'db131',
+    'c2': 'db101',
+    'c3': 'db142',
+    'c4': 'db121',
+    'c5': 'db131',
+}
+
+
+def get_db_cluster(oldwiki_db):
+    command = f"salt-ssh -E 'db131*' cmd.run 'USE mhglobal; SELECT wiki_dbcluster FROM cw_wikis WHERE wiki_dbname = \"{oldwiki_db}\"'"
+    result = subprocess.run(command, shell=True, stdout=subprocess.PIPE, text=True)
+    cluster_name = result.stdout.strip()
+    return db_clusters.get(cluster_name)
+
+
+def execute_salt_command(cluster, command):
```

Comment:
Should be typed but I recommend splitting into generate_salt and execute_salt. That way you can confirm that your salt command is valid. You could possibly then use that in other places.